agent/command/s3_put.go

```diff
@@ -387,7 +388,7 @@ retryLoop:
 	logger.Task().WarningWhen(strings.Contains(s3pc.Bucket, "."), "bucket names containing dots that are created after Sept. 30, 2020 are not guaranteed to have valid attached URLs")
 
 	if len(uploadedFiles) != len(filesList) && !s3pc.skipMissing {
-		logger.Task().Infof("%d requested, %d uploaded", len(filesList), len(uploadedFiles))
+		logger.Task().Infof("attempted to upload %d files, %d successfully uploaded", len(filesList), len(uploadedFiles))
```

Comment:
nit but feel free to reject: I think the phrase is typically something like "successfully uploaded N of M files"